src/libraries/Native/Unix/CMakeLists.txt

```diff
@@ -296,6 +296,9 @@ endif()
 add_subdirectory(System.Native)
 
 if (NOT CLR_CMAKE_PLATFORM_WASM)
+    # In coreclr we currently link to it as part of the coreclr build.
+    # For libraries and mono build we need to the shared library until mono links to it.
```

Comment:
```suggestion
    # For libraries and mono build we need to build the shared library until mono links to it.
```

Reply:
I can remove this as we changed to use `GEN_SHARED_LIB` which is set when not building in WASM for libraries.